com.github.akiraux.akira.json

```diff
@@ -0,0 +1,67 @@
+{
+   "app-id": "com.github.akiraux.akira",
+   "runtime": "org.gnome.Platform",
+   "runtime-version": "3.30",
+   "sdk": "org.gnome.Sdk",
+   "base":"io.elementary.BaseApp",
+   "base-version": "juno",
+   "command": "com.github.akiraux.akira",
+   "cleanup": [
+      "/include",
+      "/lib/pkgconfig",
+      "/lib/debug",
+      "/share/vala",
+      "/man",
+      "*.a",
+      "*.la"
+   ],
+   "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--share=network",
```

Comment:
Why we would need network access here?

Reply:
related with xmlns or similar, but we can avoid right now